src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageSearch/PackageSearchResultPackageDetailed.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using System.Text.Json.Serialization;
+using NuGet.Protocol.Core.Types;
+
+namespace NuGet.CommandLine.XPlat
+{
+    internal class PackageSearchResultPackageDetailed : PackageSearchResultPackageNormal
```

Comment:
There's double package in the name here. 

You can simplify these to 
DetailedPackageSearchResult and NormalPackageSearchResult

Reply:
renamed